source/NVDAObjects/IAccessible/chromium.py

```diff
@@ -90,21 +90,21 @@ class Document(ia2Web.Document):
 
 	def _get_treeInterceptorClass(self) -> typing.Type["TreeInterceptor"]:
 		shouldLoadVBufOnBusyFeatureFlag = bool(
-			config.conf["virtualBuffers"]["loadChromiumVbufOnBusyState"]
+			config.conf["virtualBuffers"]["loadChromiumVBufOnBusyState"]

```

Comment:
Please see this discussion: https://github.com/nvaccess/nvda/pull/13851#issuecomment-1170982580

The short version.
The case of "Vbuf" changed to a capital B. Different keys is actually an advantage: If this PR (adding the feature flag helper) is reverted, the user's config file would become invalid if the same key name were used. This would result in their config being reset to defaults.
In the case of a revert with different keys, the config will just contain a key that is no longer used.

The similar names is a bit confusing when looking at the diff, however I'm not sure this is worth trying to clarify.